src/main/cpp/t07_snake.cpp

```diff
@@ -22,6 +22,30 @@
 using namespace std;
 
 int t07_snake() {
-    
+    int n, m, num;
+	cin >> n >> m;
+	int arr[n][m];
+	num = 1;
+	for (int i = 0; i < n; i++) {
+			if (i % 2 == 0) { 
+			    for (int j = 0; j < m; j++){
+				arr[i][j] = num;
+				num++;
+			}
+			}
+			else if ( i % 2 != 0){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Condition 'i%2!=0' is always true](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4731462)